Pod/Classes/PagingMenuController.swift

```diff
@@ -324,7 +324,7 @@ extension PagingMenuController: UIScrollViewDelegate {
         move(toPage: nextPage)
     }
 
-    public func scrollViewWillBeginDecelerating(_ scrollView: UIScrollView) {
+    public func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {
```

Comment:
https://developer.apple.com/reference/uikit/uiscrollviewdelegate indicates that this should be used:

func scrollViewWillBeginDragging(UIScrollView)

when we need to detect a scroll starts.